app/javascript/flavours/glitch/styles/components/index.scss

```diff
@@ -1684,6 +1684,7 @@ noscript {
 @import 'search';
 @import 'emoji';
 @import 'doodle';
+@import 'giphy';
```

Comment:
I'm merging this into im-in.space but apparently [`giphy.scss`](https://github.com/koyuspace/mastodon/blob/main/app/javascript/flavours/glitch/styles/components/giphy.scss) is missing here.

Reply:
Added the file to the PR, thanks for clarifying